### PR TITLE
Fixing Bug #1270 and resolve all yarn lint warning.

### LINKED ISSUE #1270: [Bug] blocks.getBlockByIndex returns BlockAPI object even if block by passed index is not exists
Create new Editor.js instance with empty data

Steps to reproduce:
1. Init Editor.js with empty data
2. Call `editor.blocks.getBlockByIndex(1)`

Expected behaviour:
Warning or error should appear in console

Actual:
`getBlockByIndex` returns BlockAPI object

Editor.js version: 2.18

Plugins you use with their versions: latest

### PR DESCRIPTION
**Pull request by robonetphy**
Resolve #1270 and solve the following `yarn lint` warning : 
Screenshot (98)

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 - `Improvements` - Unuseful log about missed i18n section has been removed  [#1269](https://github.com/codex-team/editor.js/issues/1269)
 - `Fix` - Fixed issue with enter key in inputs and textareas [#920](https://github.com/codex-team/editor.js/issues/920)
 - `Improvements` - Allowed to set `false` as `toolbox` config in order to hide Toolbox button [#1221](https://github.com/codex-team/editor.js/issues/1221)
+- `Fix` - Fixed issue with getBlockByIndex API [#1270](https://github.com/codex-team/editor.js/issues/1270)
```
Comment (neSpecc) [inline docs/CHANGELOG.md:17]: what issue?

Comment (robonetphy) [reply docs/CHANGELOG.md]: It's a bug which will return the block object when no data present in EditorJS holder and we call editor.getBlockByIndex(1).

File: src/components/modules/api/blocks.ts [hunk 2]
```diff
@@ -60,6 +60,12 @@ export default class BlocksAPI extends Module {
   public getBlockByIndex(index: number): BlockAPIInterface {
     const block = this.Editor.BlockManager.getBlockByIndex(index);
 
+    if (block === undefined) {
+      _.log('Block is `undefined` at index `' + index + '`', 'error');
```
Comment (gohabereg) [inline src/components/modules/api/blocks.ts:64]: Better to just warn I think

Comment (robonetphy) [reply src/components/modules/api/blocks.ts]: Done

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +14,7 @@
 - `Improvements` - Unuseful log about missed i18n section has been removed  [#1269](https://github.com/codex-team/editor.js/issues/1269)
 - `Fix` - Fixed issue with enter key in inputs and textareas [#920](https://github.com/codex-team/editor.js/issues/920)
 - `Improvements` - Allowed to set `false` as `toolbox` config in order to hide Toolbox button [#1221](https://github.com/codex-team/editor.js/issues/1221)
+- `Fix` - Fixed bug of getBlockByIndex API[#1270](https://github.com/codex-team/editor.js/issues/1270)
```
Comment (ranemihir) [inline docs/CHANGELOG.md:17]: Maybe try writing a more expressive description. Also, put a full stop at the end and a blank before the link.

**Review by ranemihir — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Review by gohabereg — APPROVED** [review]

**Pull request merged**